server/routes/api.ts

```diff
@@ -163,27 +168,20 @@ router.post(
 
 router.post(
 	'/case/:caseId?',
+	withOktaServerSideValidation,
 	cancellationSfCasesAPI('case', 'CREATE_CANCELLATION_CASE'),
 );
 router.patch(
 	'/case/:caseId?',
+	withOktaServerSideValidation,
 	cancellationSfCasesAPI('case/:caseId', 'UPDATE_CANCELLATION_CASE', [
 		'caseId',
 	]),
 );
 
-router.post(
```

Comment:
possibly from a merge but not sure why this has been removed?

Reply:
Yeah, I think this must have been from a merge but it definitely shouldn't have been deleted! Good catch, thank you.